test.sh

```diff
@@ -155,7 +166,18 @@ fi
 #
 if [[ "$RUN" =~ "lint" ]] ; then
   start_context "test/golint"
-  [ -x "$(which golint)" ] && run golint ./...
+  check_golint() {
+    errors=$([ -x "$(which golint)" ] && golint -min_confidence=0.81 ./...)
+    if [ "x${errors}" == "x" ]; then
+      return 0
+    else
+      # Re-run to display the errors
+      [ -x "$(which golint)" ] && run golint -min_confidence=0.81 ./...
```

Comment:
whoops; commented in the wrong place before.

Anyway... I believe you should be able to simply `echo ${errors}` to get this behavior (logging the output) without having to run it again.